spec.html

```diff
@@ -8592,17 +8645,19 @@ <h1>[[Construct]] ( _argumentsList_, _newTarget_ )</h1>
     </emu-clause>
 
     <emu-clause id="sec-createbuiltinfunction" aoid="CreateBuiltinFunction">
-      <h1>CreateBuiltinFunction ( _steps_, _internalSlotsList_ [ , _realm_ [ , _prototype_ ] ] )</h1>
-      <p>The abstract operation CreateBuiltinFunction takes arguments _steps_ and _internalSlotsList_ (a List of names of internal slots) and optional arguments _realm_ and _prototype_. _internalSlotsList_ contains the names of additional internal slots that must be defined as part of the object. This operation creates a built-in function object. It performs the following steps when called:</p>
+      <h1>CreateBuiltinFunction ( _steps_ [ , _additionalInternalSlotsList_ [ , _realm_ [ , _prototype_ ] ] ] )</h1>
+      <p>The abstract operation CreateBuiltinFunction takes arguments _steps_ and optional arguments _additionalInternalSlotsList_ (a List of internal slot names), _realm_, and _prototype_. _additionalInternalSlotsList_ contains the names of additional internal slots that must be defined as part of the object. This operation creates a built-in function object. It performs the following steps when called:</p>
       <emu-alg>
         1. Assert: _steps_ is either a set of algorithm steps or other definition of a function's behaviour provided in this specification.
         1. If _realm_ is not present, set _realm_ to the current Realm Record.
         1. Assert: _realm_ is a Realm Record.
         1. If _prototype_ is not present, set _prototype_ to _realm_.[[Intrinsics]].[[%Function.prototype%]].
-        1. Let _func_ be a new built-in function object that when called performs the action described by _steps_. The new function object has internal slots whose names are the elements of _internalSlotsList_, and an [[InitialName]] internal slot.
-        1. Set _func_.[[Realm]] to _realm_.
-        1. Set _func_.[[Prototype]] to _prototype_.
+        1. Let _internalSlotsList_ be &laquo; [[Prototype]], [[Extensible]] &raquo; and the internal slots specified in <emu-xref href="#table-internal-slots-of-built-in-function-objects"></emu-xref> or <emu-xref href="#table-internal-slots-of-ecmascript-function-objects"></emu-xref> and &laquo; [[InitialName]] &raquo;, as appropriate.
```

Comment:
For exotic built-ins, this includes [[InitialName]] twice.

Reply:
So does [your version](https://github.com/tc39/ecma262/pull/2115#discussion_r493967623).

Reply:
Yes, because that suggestion wasn't attempting to resolve this problem. That's why I raised it separately.